xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -221,7 +221,7 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
           <input type="hidden" name="${xCommentClass}_author" value="$xcontext.user"/>
         #else
           <label for="${xCommentClass}_author"><input type="text" id="${xCommentClass}_author" name="${xCommentClass}_author" value="$author" maxlength="255" /> says:</label>
-          <strong><a href='$xwiki.getURL('XWiki.XWikiLogin', 'login', "xredirect=$escapetool.url($xwiki.relativeRequestURL)")'>$services.localization.render('login')</a></strong>
+          <strong><a href='$xwiki.getURL('XWiki.XWikiLogin', 'login', "xredirect=$escapetool.url($xwiki.relativeRequestURL)")'>$escapetool.html($services.localization.render('login'))</a></strong>
```

Comment:
@surli it is user input as you can overwrite the translation at runtime. The best practice IMO to is always escape translation messages when using them inside HTML, but we normally use ``$escapetool.xml`` and not ``$escapetool.html``.

Reply:
> @surli it is user input as you can overwrite the translation at runtime.

indeed, not exactly the original issue description but always better to have it fixed.

> but we normally use $escapetool.xml and not $escapetool.html

ok from what I read on the javadoc for `escapetool.xml`, it might be a bit overkill to use `escapetool.html`.